src/main/java/com/example/bookstoreproject/configuration/CloudinaryConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package com.example.bookstoreproject.configuration;
+
+import com.cloudinary.Cloudinary;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Configuration
+@PropertySource(value = {"classpath:application-dev.properties"})
+public class CloudinaryConfiguration {
+    @Value("${cloudinary.name}")
+    private String cloudinaryName;
+    @Value("${cloudinary.secret}")
+    private String cloudinarySecret;
+    @Value("${cloudinary.key}")
+    private String cloudinaryKey;
```

Comment:
Kết hợp với prefix ở trên, mình chỉ cần khai báo tên y vậy là được


```suggestion
    private String name;
    private String secret;
    private String key;
```

Reply:
DONE